src/sentry/grouping/ingest/seer.py

```diff
@@ -147,6 +154,32 @@ def _ratelimiting_enabled(event: Event, project: Project) -> bool:
     return False
 
 
+def _circuit_breaker_broken(event: Event, project: Project) -> bool:
+    circuit_broken = not seer_similarity_circuit_breaker.should_allow_request()
+
+    if circuit_broken:
+        logger.warning(
+            "should_call_seer_for_grouping.circuit_breaker_tripped",
+            extra={
+                "event_id": event.event_id,
+                "project_id": project.id,
+                **options.get("seer.similarity.circuit-breaker-config"),
+            },
+        )
+        metrics.incr(
+            "grouping.similarity.circuit_breaker_tripped",
+        )
+        metrics.incr(
+            "grouping.similarity.did_call_seer",
+            sample_rate=1.0,
+            tags={"call_made": False, "blocker": "circuit-breaker"},
+        )
+
+        return True
+
+    return False
```

Comment:
Nit: you could do `return circuit_broken`

Reply:
Yeah, you’re right - unlike the other similar functions this is patterned off of, there's only one `return True` off-ramp. Can do.